test/trade/actions/helpers/trade-test.js

```diff
@@ -0,0 +1,248 @@
+				outcomeID: '1',
+				marketType: 'categorical',
+				marketDescription: 'test categorical market',
+				outcomeName: 'Democratic'
+			},
+			feePercent: {
+				value: '0.099800399201596707'
+			}
+		},
+		'trans3': {
+			data: {
+				marketID: '0x000000000000000000000000000000000scalar1',
+				outcomeID: '1',
+				marketType: 'scalar',
+				marketDescription: 'test scalar market',
+				outcomeName: ''
+			},
+			feePercent: {
+				value: '0.95763203714451532'
+			}
+		}
+	};
+	testState.orderBooks = {
+		'0x000000000000000000000000000000000binary1': {
+			buy: {
+				'order1': {
+					id: 1,
+					price: '0.45',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order2': {
+					id: 2,
+					price: '0.45',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			},
+			sell: {
+				'order3': {
+					id: 3,
+					price: '0.4',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order4': {
+					id: 4,
+					price: '0.4',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			}
+		},
+		'0x0000000000000000000000000000categorical1': {
+			buy: {
+				'order1': {
+					id: 1,
+					price: '0.45',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order2': {
+					id: 2,
+					price: '0.45',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			},
+			sell: {
+				'order3': {
+					id: 3,
+					price: '0.4',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order4': {
+					id: 4,
+					price: '0.4',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			}
+		},
+		'0x000000000000000000000000000000000scalar1': {
+			buy: {
+				'order1': {
+					id: 1,
+					price: '45',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order2': {
+					id: 2,
+					price: '45',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			},
+			sell: {
+				'order3': {
+					id: 3,
+					price: '40',
+					outcome: '1',
+					owner: 'owner1'
+				},
+				'order4': {
+					id: 4,
+					price: '40',
+					outcome: '1',
+					owner: 'owner1'
+				}
+			}
+		}
+	};
+	const store = mockStore(testState);
+	const mockAugur = { augur: { ...augur }, abi: { ...abi }, constants: { ...constants } };
+	const mockLoadBidAsks = { loadBidsAsks: () => {} };
+	sinon.stub(mockLoadBidAsks, 'loadBidsAsks', (marketID, cb) => {
+		assert.isString(marketID, `didn't pass a marketID as a string to loadBidsAsks`);
+		cb();
+		return { type: 'LOAD_BIDS_ASKS' };
+	});
+
+	sinon.stub(mockAugur.augur, 'getParticipantSharesPurchased', (marketID, userID, outcomeID, cb) => {
+		cb(10);
+	});
+
+	sinon.stub(mockAugur.augur, 'getCashBalance', (takerAddress, cb) => {
+		cb('10000.0');
+	});
+
+	sinon.stub(mockAugur.augur, 'trade', (args) => {
+		const { max_value, max_amount, trade_ids, sender, onTradeHash, onCommitSent, onCommitSuccess, onCommitFailed, onNextBlock, onTradeSent, onTradeSuccess, onTradeFailed } = args;
+		onTradeHash({ data: 'data' });
+		onCommitSent({ data: 'data' });
+		onCommitSuccess({ gasFees: '0.01450404', hash: 'testhash', timestamp: 1500000000 });
```

Comment:
Why pass these three values to the callback, and not the others it is usually passed?